packages/rustfmt/package.yaml

```diff
@@ -11,17 +11,10 @@ categories:
   - Formatter
 
 source:
-  id: pkg:github/rust-lang/rustfmt@v1.6.0
-  asset:
-    - target: [darwin_x64, darwin_arm64]
-      file: rustfmt_macos-x86_64_{{version}}.tar.gz
-      bin: rustfmt_macos-x86_64_{{version}}/rustfmt
-    - target: linux_x64
-      file: rustfmt_linux-x86_64_{{version}}.tar.gz
-      bin: rustfmt_linux-x86_64_{{version}}/rustfmt
-    - target: win_x64
-      file: rustfmt_windows-x86_64-msvc_{{version}}.zip
-      bin: rustfmt_windows-x86_64-msvc_{{version}}/rustfmt.exe
+  id: pkg:generic/rustfmt@x
+  build:
+    run: cp $(which rustfmt) . || echo "rustfmt not found"
+    bin: rustfmt
```

Comment:
This is problematic for a couple of reasons:

1) It has a placeholder version number. Packages must refer to real version numbers.
2) The build script simply copies an external installation of `rustfmt` into the Mason installation directory. This is not a self-contained installation, and may also be problematic for loading linked libraries (relative `rpath`).
3) Doesn't support Windows (although this is an easy fix by providing a powershell script as well).